api/candidatesApiClient.ts

```diff
@@ -35,13 +38,14 @@ export async function getCandidates(page: number, pageSize: number): Promise<Lis
     }
 }
 
-export async function getCandidateById(cid: number): Promise<Candidate> {
+export async function getCandidateById(cid: number, sessionId: string): Promise<Candidate> {
```

Comment:
where is this used? we don't seem to have updated the place that calls this?
Is that now broken?

Reply:
Ahh had missed this, have gotten it working now